powerlevel9k.zsh-theme

```diff
@@ -1062,6 +1062,18 @@ prompt_symfony2_version() {
   fi
 }
 
+set_default POWERLEVEL9K_VPN_IP_INTERFACE "tun"
+# prompt if vpn active
+prompt_vpn_ip() {
+  if [[ -x /usr/bin/nmcli ]]; then
+    for vpn_iface in $(/bin/ip tuntap | grep -e ^"$POWERLEVEL9K_VPN_IP_INTERFACE" | cut -d":" -f1)
```

Comment:
Don't use the absolut path here. The path to `ip` can change, depending on OS and distribution.